plugins/plugin.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <plugin id="optimove-cordova-plugin" version="1.0.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
     <name>OptimoveSDKPlugin</name>
+    <js-module name="OptimoveCore" src="www/OptimoveCore.js"/>
     <js-module name="Optimove" src="www/Optimove.js">
```

Comment:
Do we need to declare both modules / what's the reason for the split?

Reply:
(ah sorry see this is how we have it set up in Kumulos plugin too)

Reply:
i think it's possible to output a module from webpack immediately https://webpack.js.org/configuration/output/#outputmodule . But it's an experimental feature and i figured easier to not mess up with it